src/object/tests/SConscript

```diff
@@ -11,9 +11,13 @@ def scons():
 
     unit_env.Append(RPATH_FULL=['$PREFIX/lib64/daos_srv'])
     unit_env.Append(CPPDEFINES=['-DDAOS_PMEM_BUILD'])
-    test = unit_env.d_test_program(['srv_checksum_tests.c', '../srv_csum.c'],
+    unit_env.d_test_program(['srv_checksum_tests.c', '../srv_csum.c'],
                                    LIBS=['daos_common_pmem', 'gurt', 'cmocka', 'vos', 'bio', 'abt'])
-    unit_env.Install('$PREFIX/bin/', test)
```

Comment:
probably need to fix the flake errors.   Just line the next line with first line.

Reply:
think I've fixed this